CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 Changelog
 ====
+### 1.6.14
+
+* No changes; fixing a bad release.

```

Comment:
can you indicate which release version was bad in the change log?

Reply:
Done!